### DISCUSSION TITLE
How can I install apps on here

### DISCUSSION THREAD

**Discussion by bitq1** [InfiniTimeOrg/InfiniTime#2023] [Q&A]
I want to install game of life aswell as a custom game I want to make but I'm not sure how I would upload it.

**Comment by FintasticMan** [comment]
InfiniTime is a monolithic firmware, which means that all apps and features are built into the firmware. To create apps, you'll need to make changes to InfiniTime and then build and flash it. You can do this safely over BLE if you only have a sealed device. It's not possible (yet) to upload just an app, without the rest of the firmware.

For developing an app/game, you can refer to the documentation, [here](https://github.com/InfiniTimeOrg/InfiniTime/blob/main/doc/code/Apps.md) and [here](https://github.com/InfiniTimeOrg/InfiniTime/blob/main/doc/buildAndProgram.md), as well as taking inspiration from the apps that are already included in the firmware, [here](https://github.com/InfiniTimeOrg/InfiniTime/tree/main/src/displayapp/screens).

It would also be nice if you joined [the pinetime-dev chat](https://wiki.pine64.org/wiki/Main_Page#Chat_Platforms), as it's easier to have a conversation like that.
